lisp/i18n/ts/fr_FR/lisp.ts

```diff
@@ -1500,323 +1480,313 @@
     <message>
       <location filename="../../../ui/logging/common.py" line="49"/>
       <source>Thread ID</source>
-      <translation type="unfinished">Thread ID</translation>
+      <translation>ID du fil</translation>
     </message>
     <message>
       <location filename="../../../ui/logging/common.py" line="50"/>
       <source>Thread name</source>
-      <translation type="unfinished">Thread name</translation>
+      <translation>Nom du sujet</translation>
```

Comment:
That isn't "sujet"!!!, that is "fil". But we might be better here to keep "thread" even.